src/main/kotlin/commands/RulesCommand.kt

```diff
@@ -0,0 +1,23 @@
+package commands
+
+import Command
+import RulesConfig
+import arg
+import doesLater
+import string
+
+object RulesCommand : Command("r") {
+    init {
+        string("rule") {
+            doesLater { context ->
+                val ruleName: String = context arg "rule"
+                message.channel.send(FileManager.readConfigSafe<RulesConfig>(ConfigType.RULES, false)!!.rules.getOrDefault(ruleName, "No such rule $ruleName."))
```

Comment:
don't assert non null, it will return null if the file doesn't exist. Instead, use this
```kotlin
val rule = FileManager.readConfigSafe<RulesConfig>(ConfigType.RULES, false)?.let {
    rules.getOrDefault(ruleName, "Couldn't find rule $ruleName.")
} ?: "Couldn't find rule $ruleName"
```

Reply:
Oh yeah, good point, let me change that.